lib/screen/home/home_page.dart

```diff
@@ -608,11 +608,15 @@ class HomePageState extends State<HomePage>
         .toList();
     final isRefresh =
         !listEquals(activeAddresses, NftCollectionBloc.activeAddress);
+    log.info("activeAddresses: $activeAddresses");
```

Comment:
2 similar logs, can we reduce?

Reply:
no they are different `activeAddresses` and `NftCollectionBloc.activeAddress`.

Reply:
ok